.github/workflows/build-and-release-dev.yml

```diff
@@ -0,0 +1,151 @@
+
+      - name: Build Taipy templates
+        working-directory: ./taipy/templates
+        run: python setup.py build_py && python -m build
+
+      - name: Create tag and release Taipy templates
+        working-directory: ./taipy/templates
+        run: |
+          gh release create ${{ steps.version-setup.outputs.templates_VERSION }}-templates ./dist/${{ github.event.repository.name }}-templates-${{ steps.version-setup.outputs.templates_VERSION }}.tar.gz --target ${{ steps.extract_hash.outputs.HASH }} --prerelease --title ${{ steps.version-setup.outputs.templates_VERSION }}-templates --notes "Release Draft ${{ steps.version-setup.outputs.templates_VERSION }}-templates"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install Taipy templates
+        working-directory: ./taipy/templates
+        run: |
+          pip install ./dist/${{ github.event.repository.name }}-templates-${{ steps.version-setup.outputs.templates_VERSION }}.tar.gz
+
+      - name: Build Taipy package
+        run: python setup.py build_py && python -m build
+
+      - name: Create tag and release Taipy
+        run: |
+          gh release create ${{ steps.version-setup.outputs.VERSION }} ./dist/${{ github.event.repository.name }}-${{ steps.version-setup.outputs.VERSION }}.tar.gz --target ${{ steps.extract_hash.outputs.HASH }} --prerelease --title ${{ steps.version-setup.outputs.VERSION }} --notes "Release Draft ${{ steps.version-setup.outputs.VERSION }}"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install Taipy
+        run: |
+          pip install ./dist/${{ github.event.repository.name }}-${{ steps.version-setup.outputs.VERSION }}.tar.gz
+
+      - name: Check Taipy Installation
+        run: |
+          python tools/validate_taipy_install.py
+
+      - name: Bundle all packages in main release tag
+        run: |
+          gh release upload ${{ steps.version-setup.outputs.VERSION }} ./taipy/config/dist/${{ github.event.repository.name }}-config-${{ steps.version-setup.outputs.config_VERSION }}.tar.gz --clobber
+          gh release upload ${{ steps.version-setup.outputs.VERSION }} ./taipy/core/dist/${{ github.event.repository.name }}-core-${{ steps.version-setup.outputs.core_VERSION }}.tar.gz --clobber
+          gh release upload ${{ steps.version-setup.outputs.VERSION }} ./taipy/gui/dist/${{ github.event.repository.name }}-gui-${{ steps.version-setup.outputs.gui_VERSION }}.tar.gz --clobber
+          gh release upload ${{ steps.version-setup.outputs.VERSION }} ./taipy/rest/dist/${{ github.event.repository.name }}-rest-${{ steps.version-setup.outputs.rest_VERSION }}.tar.gz --clobber
+          gh release upload ${{ steps.version-setup.outputs.VERSION }} ./taipy/templates/dist/${{ github.event.repository.name }}-templates-${{ steps.version-setup.outputs.templates_VERSION }}.tar.gz --clobber
+        env:
```

Comment:
We only have taipy dependencies config, core, gui, rest, templates. I would expect to have taipy package as well, no?

Reply:
The Taipy Package .tar.gz is already included on the version. This is adding the other packages to same release on github